stubs/CMakeLists.txt

```diff
@@ -99,6 +99,12 @@ MESSAGE("--    HDF5 Include directory: ${HDF5_INCLUDE_DIR}")
 MESSAGE("--    HDF5 Library directories: ${HDF5_LIBRARY_DIRS}")
 MESSAGE("--    HDF5 Libraries: ${HDF5_LIBRARIES}")
 
+# find Coin
+FIND_PACKAGE( COIN REQUIRED )
```

Comment:
I remove the space in my modification, and in the place to took it from :) 
> On Sep 9, 2016, at 11:39, Anthony Scopatz <notifications@github.com> wrote:
> 
> In stubs/CMakeLists.txt <https://github.com/cyclus/cyclus/pull/1282#discussion_r78210714>:
> 
> > @@ -99,6 +99,12 @@ MESSAGE("--    HDF5 Include directory: ${HDF5_INCLUDE_DIR}")
> >  MESSAGE("--    HDF5 Library directories: ${HDF5_LIBRARY_DIRS}")
> >  MESSAGE("--    HDF5 Libraries: ${HDF5_LIBRARIES}")
> >  
> > +# find Coin
> > +FIND_PACKAGE( COIN REQUIRED )
> Please don't put spaces inside of the parens.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/cyclus/cyclus/pull/1282/files/12390d1fa9dcf6822ff7adb20a5b93cfc4e4eea5#r78210714>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AOcZOitgv2fRnLTvYjIrnh1l0Lg1q6rxks5qoYvHgaJpZM4J5Ixh>.
>